aspnetcore/razor-pages/index.md

```diff
@@ -426,13 +426,13 @@ The app has the following file/folder structure:
     * *Edit.cshtml*
     * *Index.cshtml*
 
-The *Pages/Customers/Create.cshtml* and *Pages/Customers/Edit.cshtml* pages redirect to *Pages/Customers/Index.cshtml* after success. The string `./Index` is part of the URI to access the preceding page. The string `./Index` can be used to generate URIs to the *Pages/Customers/Index.cshtml* page. For example:
+The *Pages/Customers/Create.cshtml* and *Pages/Customers/Edit.cshtml* pages redirect to *Pages/Customers/Index.cshtml* after success. The string `./Index` is a relative page name used to access the preceding page. It is used to generate URLs to the *Pages/Customers/Index.cshtml* page. For example:
 
 * `Url.Page("./Index", ...)`
-* `<a asp-page="/Customers/Index">Customers Index Page</a>`
+* `<a asp-page="./Index">Customers Index Page</a>`
```

Comment:
I think referencing an absolute page here when talking about the relative `./Index` page name was a bit confusing. The text inside of the `a` tag already makes it clear which page this resolves to.